ansible/roles/vm_set/tasks/start_sonic_vm.yml

```diff
@@ -54,8 +56,14 @@
              mgmt_ip="{{ mgmt_ip_address }}/{{ mgmt_prefixlen }}"
              mgmt_gw={{ vm_mgmt_gw | default(mgmt_gw) }}
              new_password={{ sonic_password }}
+             num_asic={{ num_asic }}
   register: kickstart_output
 
 - name: Fail if kickstart gives error for {{ dut_name }}
   fail: msg="Start sonic vm weren't succesfull"
   when: kickstart_output.kickstart_code != 0
+
+- name: Reboot sonic-vm to bring up as multi-asic vs {{ num_asic }}
+  command: "virsh reboot {{ dut_name }}"
```

Comment:
not needed

Reply:
Removed as we can use a pre-built sonic-vs image where number of asics is updated in asic.conf.